boards/cubepilot/cubeorange/init/rc.board_mavlink

```diff
@@ -5,3 +5,6 @@
 
 # Start MAVLink on the USB port
 mavlink start -d /dev/ttyACM0
+
+# Start ADS-B receiver mavlink connection
+mavlink start -d /dev/ttyS4
```

Comment:
Newline is missing - sorry about the nitpick, but I'm trying to also give feedback for future PRs.

Reply:
Which newline do you mean? And nitpicking is fine if it improves the code/code style ^^

Reply:
See the red exclamation mark. The line you added does not end in a newline (but should)

Reply:
Ah, this was a windows line ending. I didn't see the new line ending. Will change it!